core/lib/spree/core/search/base.rb

```diff
@@ -17,7 +17,8 @@ def retrieve_products
           curr_page = page || 1
 
           unless Spree::Config.show_products_without_price
-            @products = @products.where("spree_prices.amount IS NOT NULL").where("spree_prices.currency" => current_currency)
+            @products = @products.where("spree_prices.amount IS NOT NULL")
+                                 .where("spree_prices.currency" => current_currency)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.